src/Apps/Order/Components/AddressVerificationFlow.tsx

```diff
@@ -130,19 +185,15 @@ const AddressVerificationFlow: React.FC<AddressVerificationFlowProps> = ({
         : AddressVerifiedBy.ARTSY
 
     if (selectedAddress) {
-      setModalType(null)
+      setShowModal(false)
       onChosenAddress(verifiedBy, selectedAddress.address, false)
     }
   }, [addressOptions, onChosenAddress, selectedAddressKey])
```

Comment:
Not sure if this is meaningful, but since they are different handlers I thought i would specify the case where the user dismisses the modal vs clicking a button to edit their address.